api/v1alpha1/nginxingresscontroller_types.go

```diff
@@ -50,6 +50,28 @@ type NginxIngressControllerSpec struct {
 	// will be from the Azure LoadBalancer annotations here https://cloud-provider-azure.sigs.k8s.io/topics/loadbalancer/#loadbalancer-annotations
 	// +optional
 	LoadBalancerAnnotations map[string]string `json:"loadBalancerAnnotations,omitempty"`
+
+	// DefaultSSLCertificate is a struct with a secret with the fields namespace and name which is used to create the ssl certificate used by the default HTTPS server
```

Comment:
this comment doesn't fully describe what this field actually configures. It's not a default HTTPS server being configured. It's the NginxIngressController being managed as a result of this CRD. We should also specify that if this field isn't defined then no DefaultSSLCertificate will be used